index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/styles.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <a href="#" class="logo"> My Logo </a>
+        <div class="menu">
+          <img src="images/svg/Menu.svg" alt="Menu" />
+        </div>
+        <ul class="nav-menu">
+          <li class="nav-item">
+            <a href="#" class="hover-blue-text"> Portfolio </a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="hover-blue-text"> About </a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="hover-blue-text"> Contact </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="header">
+      <div class="container">
+        <h1 class="title">
+          I'm Eduardo <br />
+          Glad to see you!
+        </h1>
+        <p class="description">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hesitate to contact
+          me.
+        </p>
+        <a href="#" class="connect-button hover-blue-text">Let's connect</a>
+        <div class="rrss">
```

Comment:
[OPTIONAL]
- It is important to always use descriptive names for your classes and id, this was anyone going through your code knows what and how everything works. :+1: